packages/apollo-engine-reporting/src/extension.ts

```diff
@@ -157,7 +163,12 @@ export class EngineReportingExtension<TContext = any>
       }
     }
 
-    if (this.options.privateVariables !== true && o.variables) {
+    // Default: all variables are private; == checks for both null and undefined
+    if (
+      (!this.options.privateVariables == null ||
```

Comment:
I think this should be an `&&` (it is noticeably different from the other one above).

I think they can both be simplified as `if (![null, undefined, true].contains(this.options.privateWhatever))`.

Reply:
Typing made that a real hassle, but I can make it work.